installation/db_migration/_migrate_1.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE artefact_metadata ADD COLUMN new_id CHAR(32);
```

Comment:
don't we need to guard against the column to be already present?

Reply:
The script should be only executed if the migration did not happen yet, and in that case the column does not exist yet. If the script is executed anyways, that can be considered an error and hence it also makes sense that this query will raise an exception.